### PR TITLE
[coin-integration] Linea

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

Support of [linea](https://linea.build/): A developer-ready zkEVM rollup for scaling Ethereum dapps

Screenshot 2024-01-03 at 18 21 23

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/LIVE-10567

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/live-config/src/featureFlags/defaultFeatures.ts [hunk 1]
```diff
@@ -66,6 +66,8 @@ export const CURRENCY_DEFAULT_FEATURES = {
   currencyVelasEvm: DEFAULT_FEATURE,
   currencyCasper: DEFAULT_FEATURE,
   currencyNeonEvm: DEFAULT_FEATURE,
+  currencyLinea: DEFAULT_FEATURE,
+  currencyLineaGoerli: DEFAULT_FEATURE,
```
Comment (hedi-edelbloute) [inline libs/live-config/src/featureFlags/defaultFeatures.ts:70]: To be discussed but shouldn't we cover both linea and lineagoerli with the same feature flag ?

**Review by hedi-edelbloute — CHANGES_REQUESTED** [review]
CI to fix : desktop and mobile code checks, test docs, test libraries (coin-evm) failing

Rest is fine :+1:

File: libs/live-config/src/featureFlags/defaultFeatures.ts (hunk 1 above)
Comment (lvndry) [reply libs/live-config/src/featureFlags/defaultFeatures.ts:70]: For the currencies currently supporting a testnet they have a different feature flag. I think it's better to have seperate feature_flag as goerli is only used in developer mode, its overridden locally by the developer when the mainnet is set in firebase for the users

Comment (hedi-edelbloute) [reply libs/live-config/src/featureFlags/defaultFeatures.ts:70]: "feature_flag as goerli is only used in developer mode"

=> that's why i'm asking, we are already out of the standard usage

**Pull request merged**